### PR TITLE
Fix a typo, add missing space in code snippet

### PR DESCRIPTION
**Pull request by Abrasha**
Fixes https://github.com/scala/scala-lang/issues/675

### CODE REVIEW

File: tutorials/tour/_posts/2017-02-13-traits.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ iterator.next()  // prints 1
 This `IntIterator` class takes a parameter `to` as an upper bound. It `extends Iterator[Int]` which means that the `next` method must return an Int.
 
 ## Subtyping
-Subtypes of traits can be used where a the trait is required.
+Subtypes of traits can be used where a trait is required.
```
Comment (som-snytt) [inline tutorials/tour/_posts/2017-02-13-traits.md]: How about:
```
Where a given trait is required, a subtype of the trait can be used instead.
```
Avoids the forward reference.

Comment (Abrasha) [inline tutorials/tour/_posts/2017-02-13-traits.md]: Cool! Will update PL this evening (in 10 hours)

Comment (Abrasha) [inline tutorials/tour/_posts/2017-02-13-traits.md]: @som-snytt done. Please review.

**Comment by som-snytt** [comment]
LGTM thx.

**Comment by heathermiller** [comment]
Thanks @Abrasha and @som-snytt!

**Pull request merged**